examples/demo_mult_image_sockets.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
```

Comment:
This example seems to have some performance issues on my computer (pan/zoom become slow) and it is not quite clear to me what it does. Doesn't seem related to vectors, and also seems to be a bit to advanced functionality (i.e. custom Qt Signals/Slots for our current examples. Maybe you keep a copy of this locally and then in the future if there is more of an advanced use-case section we could highlight this.

Reply:
Running it again I see the timed updates, but still think this feature doesn't belong in the examples at this point

Reply:
also removed